cloudferrylib/os/network/neutron.py

```diff
@@ -325,7 +325,7 @@ def convert(neutron_object, cloud, obj_name):
     def convert_networks(self, net, cloud):
         identity_res = cloud.resources[utl.IDENTITY_RESOURCE]
         net_res = cloud.resources[utl.NETWORK_RESOURCE]
-        get_tenant_name = identity_res.get_tenants_func()
+        get_tenant_name = identity_res.get_tenants_func(default_tenant='admin')
```

Comment:
This change will not be required if `default_tenant` value is dynamically picked from `get_tenants_func()`